Wake On Lan/Modules/AboutScreen/Tests/AboutScreenTests/AboutScreenInteractorTests.swift

```diff
@@ -0,0 +1,36 @@
+import XCTest
+
+@testable import AboutScreen
+@testable import WOLResources
```

Comment:
No, actually we do not need to mark `WOLResources` with `@testable` keyword. `WOLResources` should be included in each testable bundle.

Reply:
We have to, as BundleInfo initializer is internal